common.yml

```diff
@@ -587,7 +584,7 @@ functions:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
         local_files_include_filter:
-          - src/github.com/mongodb/mongo-tools/mongodb-cli-tools-*
+          - src/github.com/mongodb/mongo-tools/release.*
```

Comment:
Since the same artifact may get uploaded as multiple different names (for example, as a "stable" and a "latest stable" release), it is easier and less confusing to just use release.tgz (and release.msi, etc) here.